privacy_evaluator/attacks/membership_inference/membership_inference.py

```diff
@@ -67,17 +69,13 @@ def attack_output(self, x: np.ndarray, y: np.ndarray, y_attack: np.ndarray) -> D
         test_accuracy = accuracy(self.y_test, self.target_model.predict(self.x_test))
         y_attack_prediction = self.attack(x, y)
 
-        return {
-            "target_model_train_accuracy": train_accuracy,
-            "target_model_test_accuracy": test_accuracy,
-            "target_model_train_to_test_accuracy_gap": train_to_test_accuracy_gap(
-                train_accuracy, test_accuracy
-            ),
-            "target_model_train_to_test_accuracy_ratio": train_to_test_accuracy_ratio(
-                train_accuracy, test_accuracy
-            ),
-            "attack_model_accuracy": accuracy(y_attack, y_attack_prediction),
-        }
+        return UserOutputInferenceAttack(
+            train_accuracy,
+            test_accuracy,
+            train_to_test_accuracy_gap(train_accuracy, test_accuracy),
+            train_to_test_accuracy_ratio(train_accuracy, test_accuracy),
+            accuracy(y_attack, y_attack_prediction),
+        )
```

Comment:
In the issue one requested change was also to not calculate these static/fixed values over and over again (the first four values are actually independent of the attack and thus stay the same for each performed attack). But I would not change anything, for now, we can just keep that in mind for the next sprint.

Reply:
yes thats right let's keept that in mind